spec/models/generic_file_spec.rb

```diff
@@ -153,7 +153,8 @@
       expect(local[Solrizer.solr_name("mime_type")]).to eq ["image/jpeg"]
       expect(local['all_text_timv']).to eq('abcxyz')
       expect(local[Solrizer.solr_name('belongsToCommunity')]).to eq [community.id]
-      expect(local[Solrizer.solr_name('doi')]).to eq ['doi:10.5072/FKEXAMPLE']
+      expect(local[Solrizer.solr_name('doi', :symbol)]).to eq ['doi:10.5072/FKEXAMPLE']
```

Comment:
I think it's a good idea to use the solr_name helper to hide the configuration details. Those suffixes are all dynamic fields from the Solr schema.xml. Unfortunately, Solrizer isn't exhaustive as you've noticed.

The  reason for having both fields is that one is used for facets and the other is used for search and display. Checking for both of them would just be verifying the catalog_controller, but maybe we should?

```*_sim``` is [string not stored](https://github.com/ualbertalib/HydraNorth/blob/master/solr_conf/conf/schema.xml#L74)